### ISSUE TITLE
A11y: Employee Management - lacking descriptive page titles

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5507]
All pages must have a {{<title>}} that describe topic or purpose. Current storefront pages follow this pattern: 
{{<name of page> | Demostore}}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39644**



[created from NEXT-39644, comment 513184]
